sw/device/tests/aon_timer_wdog_lc_escalate_test.c

```diff
@@ -34,16 +34,15 @@ const test_config_t kTestConfig;
  * wdog is programed to bark.
  */
 enum {
-  kWdogBarkMicros = 3 * 1000,          // 3 ms
-  kWdogBiteMicros = 4 * 1000,          // 4 ms
-  kEscalationPhase0Micros = 1 * 1000,  // 1 ms
-  // The cpu value is slightly larger as the busy_spin_micros
-  // routine cycle count comes out slightly smaller due to the
-  // fact that it does not divide by exactly 1M
-  // see sw/device/lib/runtime/hart.c
-  kEscalationPhase0MicrosCpu = kEscalationPhase0Micros + 200,  // 1.2 ms
-  kEscalationPhase1Micros = 5 * 1000,                          // 5 ms
-  kEscalationPhase2Micros = 500,                               // 500 us
+  kWdogBarkMicros = 3 * 100,          // 300 us
```

Comment:
ah thank you, order of magnitude reduction!

Reply:
@matutem, timing parameter reduction should be helpful. 
Can you help me to learn the system spec such as range of these parameters, etc..?